service/rpc/share.go

```diff
@@ -45,6 +47,12 @@ func (h *Handler) handleSharesByNamespaceRequest(w http.ResponseWriter, r *http.
 		writeError(w, http.StatusBadRequest, namespacedSharesEndpoint, err)
 		return
 	}
+	// sanity check nID size
+	if len(nID) != ipld.NamespaceSize {
+		writeError(w, http.StatusBadRequest, namespacedSharesEndpoint,
+			fmt.Errorf("namespace id must be default size %d", ipld.NamespaceSize))
```

Comment:
"default" size isn't the most descriptive, since there's no non-default.

Reply:
changed err in this commit https://github.com/celestiaorg/celestia-node/pull/723/commits/377bb08cea2feb2ad02abb7b97f4d46321024a34